Doc/whatsnew/3.13.rst

```diff
@@ -354,6 +354,11 @@ disabled with the :envvar:`PYTHON_GIL` environment variable or the
 pip 24.1 or newer is required to install packages with C extensions in the
 free-threaded build.
 
+This project has been contributed by many people and the community,
+including Brett Simmers, Dino Viehland, Donghee Na, Itamar Oren, Ken Jin,
+Matt Page, and Sam Gross.
+Meta has supported in the form of many engineer-years to this project.
```

Comment:
Wording suggestions (on mobile so I can't easily check line wrapping)

```suggestion
This work is thanks to many people and organisations, including the large community of contributors
to Python and third party projects to test and enable free-threading support.
This includes, but is not limited to:
Sam Gross, Ken Jin, Donghee Na, Itamar Oren, Matt Page, Brett Simmers, and Dino Viehland.
Many of these contributors are employed by Meta Inc, which has provided significant manpower to support this project.
```

Reply:
Thanks, it's more better, and @itamaro is going to add more detail by next week :)